apps/tutorial-servo-can-control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,31 @@
+#include "TutorialServo.h"
+
+#include <cstdlib>
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees = 180.0, float minPulsewidthInMs = 1,
+                             float maxPulsewidthInMs = 2) {
+  m_servoRangeInDegrees = servoRangeInDegrees;
+  m_minPulsewidthInMs   = minPulsewidthInMs;
+  m_maxPulsewidthInMs   = maxPulsewidthInMs;
+  m_servoPwmOut(servoPin);
```

Comment:
Put these in a member initializer list. https://en.cppreference.com/w/cpp/language/constructor